tests/framework/test_engine.py

```diff
@@ -331,7 +331,7 @@ def test_SimulationContext_report_write(base_config, components, tmpdir):
     ]
     finished_sim = run_simulation(base_config, components, configuration)
     # Check for expected results written
-    results_list = [file for file in results_root.rglob("*")]
+    results_list = [file for file in results_root.rglob("*.parquet")]
```

Comment:
I had to specify `.parquet` b/c now we also have the model spec in the results dir. (I of course could've added that to the expected list instead)

Reply:
I thought you said we are already writing out a model spec file. How was that being ignored?

Reply:
@rmudambi Because that version of the model spec is [copied in the cli.py](https://github.com/ihmeuw/vivarium/blob/main/src/vivarium/interface/cli.py#L134) which this test side-steps.